content/F2015/F15-event-dropbox-coffee.md

```diff
@@ -0,0 +1,18 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-5
+Category: Events
+Tags: social, dropbox
+Slug:  dropbox-coffee-chats
+Author: Anna Lorimer
+Summary:Come and grab some coffee and snacks and chat with some female engineers
+ from Dropbox!
```

Comment:
This needs to be on the same line as the rest of the summary, or it won't parse correctly!